### PR TITLE
Fixed visual bug with Modlist export

### LINKED ISSUE #1781: Visual layout bug in mod list export view
### Operating System

Linux

### Version of Prism Launcher

Prism Launcher 8.0

### Version of Qt

Qt 6.5.2

### Description of bug

When opening the window to export a modpack's mod list, the view looks like this (it is correct):
export_mod_list_correct_1
When switching to a custom export format, it looks like this (also correct):
correct_export_mod_list_2
But, after you switch to the custom export type, if you switch to any other one, the buttons become messed up (they become very spread apart):
incorrect_export_mod_list_3
If you switch back to the custom mode, the problem persists:
incorrect_export_mod_list_4
And the problem persists until you close the export window.

### Steps to reproduce

From the main screen:
1. Select Export -> Mod List
2. Switch the export type to *Custom*
3. Switch the export type to any other option (HTML, Markdown, Plaintext, CSV, JSON)

Now the buttons in the "Optional Info" box are messed up until you close the export window.

### Suspected cause

_No response_

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

### PR DESCRIPTION
**Pull request by Trial97**
Parent PR: #1246
fixes #1781

### CODE REVIEW

File: launcher/ui/dialogs/ExportToModListDialog.ui [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <x>0</x>
     <y>0</y>
     <width>650</width>
-    <height>446</height>
+    <height>522</height>
```
Comment (TheKodeToad) [inline launcher/ui/dialogs/ExportToModListDialog.ui:10]: Did this need change?

Comment (Trial97) [reply launcher/ui/dialogs/ExportToModListDialog.ui:10]: I used QtDesigner and did the change automatically.
I can remove it if you want.

Comment (TheKodeToad) [reply launcher/ui/dialogs/ExportToModListDialog.ui:10]: You didn't accidentally resize the window then? it just changed automatically?

Comment (Trial97) [reply launcher/ui/dialogs/ExportToModListDialog.ui:10]: yes, I only played with the Size constrains of a few elements

**Review by Scrumplex — APPROVED** [review]

**Review by TheKodeToad — APPROVED** [review]

**Pull request merged**